markdown/chapters/meandeck/homework.md

```diff
@@ -0,0 +1,684 @@
+---
+authors: Doishy
+order: 7
+title: Practice Scenarios
+---
+
+## Table of Contents
+
+## Get Started
+
+Now that you've read up on the theory of how to build piles, how to play around
+opponents and how to put your deck together it is time to start putting it all
+into practice. This chapter will help you start out by trying to present some
+tricky real world scenarios and possible ways to play through them as well as
+give some ideas as to what you can do to prepare for playing.
+
+## Building/Choosing a Deck List
+
+The first step to being able to practice with the deck is to first obtain a deck
+list. Even following the precepts in the deck construction chapter, this is very
+hard to do in isolation so it's a good idea to find some inspiration first. 
+There are plenty of sites online that publish league lists, high tournament
+finishes and so on. There is also an active community, many who are happy to
+share their current lists and their thoughts/feelings on them. 
+
+Sometimes the metagame will dictate a favourite in the various styles of the 
+deck but a lot of the time there will be no "best list" to run. Simply a large
+variety of ways you could construct your 75 cards. 
+
+### Adapting Online Lists
+
+Once you've found a list you like the look of, it's time for some tuning. Maybe
+you saw something top 8 a Sunday Challenge or perhaps you really liked a brew
+mentioned in a video. Whatever it is, get your deck-building tool of choice and
+start iterating it to your own. 
```

Comment:
Maybe should be "Whatever it is, get your deck-building tool of choice and start iterating it to your own **tastes**."